src/pyodide.js

```diff
@@ -432,6 +432,15 @@ globalThis.loadPyodide = async function(config = {}) {
       false; // we preload wasm using the built in plugin now
   Module.preloadedWasm = {};
 
+  /**
+   * Tell Pyodide about an available copy of Comlink.
+   * Necessary to enable use of Comlink Proxies from Python.
+   * @private
+   */
+  Module.registerComlink = function(Comlink){
```

Comment:
It seems like we could use some sort of optional features system: there are three different packages that can lead to special interop when present:
* Comlink
* ndarray https://github.com/scijs/ndarray -- if present we could automatically fill in shape and strides when converting to Python rather than forcing user to use `np.as_array` and then do a manual cast.
* Stack trace js https://www.stacktracejs.com/ -- if present, when combined with #1447 we can produce more informative tracebacks.